homeassistant/core.py

```diff
@@ -1042,10 +1043,13 @@ def call(self, domain: str, service: str,
         This method is a coroutine.
         """
         context = context or Context()
+        self._call_id += 1
+        call_id = self._call_id
```

Comment:
We should use uuid4. Because we can restore the context object and the caller ID is not equal anymore.

Reply:
the call ID is not part of the context, it's only used by the service registry during run time?